### ISSUE TITLE
Fix: a11y - Keyboard unnavigable field resize 546

### ISSUE THREAD

**Issue by jpante** [Workday/canvas-kit#365]
## Bug Report

Cannot keyboard navigate to drag and drop resize tool

## Expected Behavior

 Can keyboard navigate to drag and drop resize tool and adjust dimensions of text area,

**Comment by jpante** [comment]
Closing issue, this is expected behavior.
